docs/Developers/Testing_coala_change_on_Bears.rst

```diff
@@ -0,0 +1,78 @@
+Testing changes to coala on coala-bears
+=======================================
+
+When large changes are made to the coala core, they should be tested
+on the coala-bears to confirm that they work as expected and don't
+break anything on the bears' side.
+
+There was an attempt to automate this through
+`#4424 <https://github.com/coala/coala/pull/4424>`_ which used to depend
+upon docker image, but unfortunately the image breaks stuff for now.
+
+But we can still do it manually, by changing the coala dependency for the
+bears to be the one consisting of the changes made. For this, we need to create
+a new branch on coala-bears, update the coala requirement to point to our
+changes and run it on CI.
+
+Set up a new testing branch
+---------------------------
+
+Assuming we have already made changes to coala through a new branch, we can
+use this branch to install coala instead of master by editing requirements.txt
+in coala-bears. Let's set-up a new branch on coala-bears for these
+changes.
+
+If you have not already, clone your fork by running:
+
+::
+
+    $ git clone https://github.com/<your-username>/coala-bears
```

Comment:
this contradicts the setup at http://api.coala.io/en/latest/Developers/Newcomers_Guide.html

clone main repo, add remote for fork.

i.e. remote `origin` should point to the central repo.